README.md

```diff
@@ -1,40 +1,25 @@
 # Single Page App Landing Page module for Drupal
 
-This module provides a standardised way for site managers to configure and serve
-single-page applications as pages in a Drupal site.
+This module provides a way for site managers to configure and serve single-page applications as pages in a Drupal site.
```

Comment:
why make this change? it goes against coding standards to have more than 80 characters per line

Reply:
or is this just that you've had a merge conflict, and have accepted the wrong set of changes?